docs/scenarios_authoring/DataSourcesAndSinks.md

```diff
@@ -34,8 +34,23 @@ The `kafka` sink configuration form will show a list of fields defined in Schema
 
 ![Kafka sink](img/kafkaSink.png "Kafka sink")
 
-If you prefer to define the whole output event as one expression containing value to be written to the Kafka topic, instead of filling separete autogenerated fields, you can do that by switching `Raw editor` to `true`.
+If you prefer to define the whole output event as one expression containing value to be written to the Kafka topic, instead of filling separate autogenerated fields, you can do that by switching `Raw editor` to `true`.
```

Comment:
---> _If, instead of filling separate ...., you prefer to .. , you can do that ..._

Reply:
Hmm, it's not part of my changes, but I'll try to change ;)